link-hint.el

```diff
@@ -59,6 +59,7 @@
     link-hint-gnus-w3m-url
     link-hint-gnus-w3m-image-url
     link-hint-deadgrep
+    link-hint-dictionary-button
```

Comment:
Could you also add a bullet in the readme for the new type?

Reply:
I've mentioned Dictionary mode in the bullet for 'buttons', which is where Woman and Man modes are currently listed.